include/dwa_planner/dwa_planner.hpp

```diff
@@ -448,31 +478,32 @@ class DWAPlanner
   int local_map_not_subscribe_count_;
   int scan_not_subscribe_count_;
 
-  ros::NodeHandle nh_;
-  ros::NodeHandle local_nh_;
-  ros::Publisher velocity_pub_;
-  ros::Publisher candidate_trajectories_pub_;
-  ros::Publisher selected_trajectory_pub_;
-  ros::Publisher predict_footprints_pub_;
-  ros::Publisher finish_flag_pub_;
-  ros::Subscriber dist_to_goal_th_sub_;
-  ros::Subscriber edge_on_global_path_sub_;
-  ros::Subscriber footprint_sub_;
-  ros::Subscriber goal_sub_;
-  ros::Subscriber local_map_sub_;
-  ros::Subscriber odom_sub_;
-  ros::Subscriber scan_sub_;
-  ros::Subscriber target_velocity_sub_;
-
-  geometry_msgs::Twist current_cmd_vel_;
-  std::optional<geometry_msgs::PoseStamped> goal_msg_;
-  geometry_msgs::PoseArray obs_list_;
-  std::optional<geometry_msgs::PolygonStamped> footprint_;
-  std::optional<nav_msgs::Path> edge_points_on_path_;
-
-  std_msgs::Bool has_finished_;
-
-  tf::TransformListener listener_;
+  rclcpp::Publisher<geometry_msgs::msg::Twist>::SharedPtr velocity_pub_;
+  rclcpp::Publisher<visualization_msgs::msg::MarkerArray>
+  ::SharedPtr candidate_trajectories_pub_;
```

Comment:
```suggestion
  rclcpp::Publisher<visualization_msgs::msg::MarkerArray>::SharedPtr candidate_trajectories_pub_;
```
uncrustifyに直されているのだと思いますが、改行位置をもう少しどうにかしたいです(せめて改行後はインデントを下げたい)